### ISSUE TITLE
Review ios backup testcase

### ISSUE THREAD

**Issue by commjoen** [OWASP/owasp-mstg#1256]
Let's review the iOS backup testcase as iTunes and iCloud backups have changed.

**Comment by sushi2k** [comment]
The backup you can open in Finder, but for some reason you cannot navigate in the terminal to it. 

```bash
roo# ls -alh Mobile*
ls: MobileSync: Operation not permitted

root# id
uid=0(root) gid=0(wheel) groups=0(wheel),1(daemon),2(kmem),3(sys),4(tty),5(operator),8(procview),9(procmod),12(everyone),20(staff),29(certusers),61(localaccounts),80(admin),701(com.apple.sharepoint.group.1),702(com.apple.sharepoint.group.2),33(_appstore),98(_lpadmin),100(_lpoperator),204(_developer),250(_analyticsusers),395(com.apple.access_ftp),398(com.apple.access_screensharing),399(com.apple.access_ssh)

root# pwd
/Users/sven/Library/Application Support
```

Is it the same on your mac?  Any idea why this is not possible, even as root?

**Comment by sushi2k** [comment]
Ok. Wasn't aware of this. Once iTerm has full disk access I can also navigate to MobileSync :-) 

http://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/

**Comment by commjoen** [comment]
Fixed in #1258
